<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GeneralSettingsView</class>
 <widget class="QWidget" name="GeneralSettingsView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="dmrIDBox">
     <property name="title">
      <string>DMR Radio ID</string>
     </property>
     <layout class="QFormLayout" name="formLayout_2">
      <item row="0" column="0">
       <widget class="QLabel" name="dMRIDLabel">
        <property name="text">
         <string>DMR ID</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="nameLabel">
        <property name="text">
         <string>Radio Name</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="radioName">
        <property name="toolTip">
         <string>Enter your name or callsign.</string>
        </property>
        <property name="accessibleName">
         <string>Name/Callsign</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="dmrID"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="bootSettingsBox">
     <property name="title">
      <string>Boot Settings</string>
     </property>
     <layout class="QFormLayout" name="formLayout_4">
      <item row="0" column="0">
       <widget class="QLabel" name="introLine1Label">
        <property name="text">
         <string>Intro Line 1</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="introLine1">
        <property name="toolTip">
         <string>First greeting line (if supported by the radio).</string>
        </property>
        <property name="accessibleName">
         <string>Intro line 1</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="introLine2Label">
        <property name="text">
         <string>Intro Line 2</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="introLine2">
        <property name="toolTip">
         <string>Second greeting line (if supported by the radio).</string>
        </property>
        <property name="statusTip">
         <string/>
        </property>
        <property name="accessibleName">
         <string>Intro line 2</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="audioSettingsBox">
     <property name="title">
      <string>Audio Settings</string>
     </property>
     <layout class="QFormLayout" name="formLayout_5">
      <item row="0" column="0">
       <widget class="QLabel" name="micLabel">
        <property name="text">
         <string>MIC Amp.</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QSpinBox" name="mic">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>10</number>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="speechLabel">
        <property name="text">
         <string>Speech Synthesis</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QCheckBox" name="speech">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
